### DISCUSSION TITLE
Excluding T cell receptor genes from Seurat object or integrated Seurat object?

### DISCUSSION THREAD

**Discussion by nilsrudqvist** [satijalab/seurat#6129] [Q&A]
Hi all,

Thanks so much for creating Seurat for us to use. 

I have several T cell 5' GEX libraries and a lot of my variable features and genes defining clusters are T cell receptor genes (TRB and TRA genes). For example:

<img width="521" alt="image" src="https://user-images.githubusercontent.com/20443080/176248538-c65678ee-48c2-4ed5-a789-727cf0bcf241.png">

I would like to remove TRB and TRA genes at some point, but not sure _when_. Should I remove these genes before integrating my libraries or can I exclude genes when calling variable features? I could not find this information when searching the forum nor using google.

Thanks in advance!
Nils

**Comment by prachi-sao** [comment]
Hello, did you determine what is optimal? Which is preferable, keeping TCRs or getting rid of them? What effect did this have on clustering?
